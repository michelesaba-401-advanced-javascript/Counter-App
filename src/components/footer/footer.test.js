import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '.';

describe('Footer Component', () => {
  it('matches snapshot without props', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('can shallowly render without props', () => {
    let footer = shallow(<Footer />);
    expect(footer.find('p').exists()).toBe(true);
  })
});
