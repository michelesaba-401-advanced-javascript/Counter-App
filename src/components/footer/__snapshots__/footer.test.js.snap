// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component matches snapshot without props 1`] = `
<footer>
  <p>
    Â© 2018 Code Fellows
  </p>
</footer>
`;
